src/witan/send/utils.clj

```diff
@@ -19,3 +19,45 @@
                                vec
                                (sort-by (apply juxt col-key))
                                ds/dataset)))
+
+(defn probs-1 [dataset]
+  (reduce merge (map (fn [{:keys [state probability]}] {state probability})
+                     (-> dataset
+                         (ds/select-columns [:state :probability])
+                         ds/row-maps))))
+
+(defn trans-mat [probabilities]
```

Comment:
Could this function 'build' the map by reducing over the States sequence in schema? 
https://github.com/MastodonC/witan.send/pull/9/files#diff-eb5f87c92704c5d4a503be087cc2aee1R33